main.tf

```diff
@@ -36,7 +36,7 @@ data "google_organization" "org" {
 
 module "seed_project" {
   source                      = "terraform-google-modules/project-factory/google"
-  version                     = "~> 5.0"
+  version                     = "~> 7.0"
```

Comment:
We should also update the project factory for cloudbuild in modules/cloudbuild/main.tf

Reply:
We can, but I don't think there's any *need* to do so.